specs/GenesisConfig/StateRecord.md

```diff
@@ -0,0 +1,131 @@
+# StateRecord
+
+Enum that describes one of the records in the state storage.
+
+## Account
+
+_type: Unnamed struct_
+
+Record that contains account information for a given account ID.
+
+### account_id
+
+_type: AccountId_
+
+The account ID of the account.
+
+### account
+
+_type: [AccountView]_
+
+The account structure in JSON-friendly format. See [Account](../DataStructures/Account.md) for details.
+
+
+## Data
+
+_type: Unnamed struct_
```

Comment:
What is `Unnamed struct` here? Why not give the struct information?

Reply:
It's Rust concept meaning a struct without a name. All fields of this structure are listed below.